src/Config.hs

```diff
@@ -279,6 +284,11 @@ optionsParser envFlags envVars = Options
       <> readValueFromEnvWithDefault "VAULTENV_LOG_LEVEL" Error envVars
       <> help ("Log-level to run vaultenv under. Options: 'error' or 'info'. " ++
                "Defaults to 'error'. Also configurable via VAULTENV_LOG_LEVEL")
+    searchPath
+      =  flag (efInheritEnv envFlags) True
+      $  long "search-path"
+      <> help ("Search PATH for the executable to call. Default: " ++
```

Comment:
s/for the executable to call/for the executable that vaultenv should call

(I was briefly confused if this was about the `vaultenv` executable itself or about the executable that `vaultenv` will exec, after putting the secrets in the environment)

Reply:
Done